examples/progress-dashboard/app.py

```diff
@@ -0,0 +1,205 @@
+# app dashboard from https://huggingface.co/spaces/davanstrien/argilla-progress/blob/main/app.py
```

Comment:
I think we could add a duplicate space button too.https://www.gradio.app/docs/gradio/duplicatebutton